### DISCUSSION TITLE
Add additional Views

### DISCUSSION THREAD

**Discussion by assemblystudio** [DigiLive/mushroom-strategy#128] [Ideas]
Is it possible to add an additional view for say Media - where all media player cards can be kept?

**Comment by DigiLive** [comment]
You can use the `extra_views` option for it.
Take a look at the last part of the [Full Example](https://github.com/AalianKhan/mushroom-strategy/wiki/Full-Example).

Or you can make a [Feature Request](https://github.com/AalianKhan/mushroom-strategy/issues/new?assignees=&labels=enhancement&projects=&template=feature_request.md&title=) to include media-players as a pre-build view.
